File: Introduction/speed-loop.c
#include <stdio.h>
#include <time.h>
int main() {
  int cycles = 1000000000;
  clock_t start, end;
  double duration;
  start = clock();
  for (int i = 0; i < cycles; i++) {
    // Empty
  }
  end = clock();
  duration = ((double)(end - start));
  printf("C looped %d times in %lf seconds\n", cycles, duration);
  return 0;
}

File: Introduction/speed-loop.js
/**
 * Created by andrewhenderson on 2/11/18.
 */
function main() {
  const cycles = 1000000000;
  let start = Date.now();
  for (let i = 0; i < cycles; i++) {
    // empty loop
  }
  let end = Date.now();
  let duration = (end - start) / 1000;
  console.log('JavaScript looped %d times in %d seconds', cycles, duration);
}
main();